packages/devtools_app/lib/src/screens/memory/panes/control/settings_dialog.dart

```diff
@@ -35,7 +35,8 @@ class MemorySettingsDialog extends StatelessWidget {
           children: [
             CheckboxSetting(
               notifier: preferences.memory.androidCollectionEnabled,
-              title: 'Show Android memory chart',
+              title:
+                  'Show Android memory chart in addition to Dart memory chart',
```

Comment:
Is this a tooltip?

Reply:
Nope, it is label for the checkbox in the settings dialog.